src/components/RequestRepository/index.tsx

```diff
@@ -50,14 +53,14 @@ const RequestRepository = () => {
     }
   };
 
-  const requesterNameColumn = {
+  const requesterNameAndComponentColumn = {
     Header: t('intake:fields.requester'),
-    accessor: 'requester.name'
-  };
-
-  const requesterComponentColumn = {
-    Header: t('intake:fields.component'),
-    accessor: 'requester.component'
+    accessor: 'requester',
+    Cell: ({ value }: any) => {
+      // Display both the requester name and the acronym of their component
+      // TODO: might be better to just save the component's acronym in the intake?
+      return `${value.name}, ${getAcronymForComponent(value.component)}`;
```

Comment:
If I'm not mistaken, the database stores the acronym, not the full name of the component. So we shouldn't have to muck around with this, we should just be able to do `value.component`.

Reply:
I'm not sure if i'm looking at this in the wrong way or missing something but it looks like the component field is the full spelled out component name. I printed out value.component and it prints out the full name. I couldn't find any representation in the DB for the acronym but I could be wrong. 

This is what the component field looks like when I access it through pgcli
![image](https://user-images.githubusercontent.com/68014929/109698452-6a44c980-7b5d-11eb-8728-2d0ef2ec1d9d.png)

Reply:
Yes, I'm mistaken. I think your solution is good then. One thing that gives me a little pause is that the database doesn't enforce that the field be one of the values we expect, it's just a text field. So if some gobbledygook got in the database, we might not find an associated acronym for it on the frontend.

Reply:
Yeah that makes sense, thats part of the reason I didn't love the solution but can't think of anything better besides storing the acronym in the DB as well.